lib/netutil/conn_pool_test.go

```diff
@@ -59,3 +60,30 @@ func testConnPoolStartStop(t *testing.T, name string, ms *metrics.Set) {
 		}
 	}
 }
+
+func TestGetPutDialConnectionPool(t *testing.T) {
+	t.Skip()
+	ms := metrics.NewSet()
+	dialTimeout := 5 * time.Second
+	compressLevel := 1
+	cp := NewConnPool(ms, "test-pool", "127.0.0.1:8401", handshake.VMSelectClient, compressLevel, dialTimeout, 0)
```

Comment:
I couldn’t find where the listener is started on port 8481. Could you point me to it?

Reply:
Hi. I ran this test locally with my vmstorage running and listening to `:8401`.

When testing this one, I added a few `println` to know if these codes really work. That's a "manually test". I didn't find a good way to build it like other unit test cases.